docs/data/material/getting-started/learn/learn.md

```diff
@@ -1,61 +1,65 @@
-Like any unfamiliar technology, MUI does have a learning curve. With practice and some patience, you will soon get the hang of it.
+## Your first component
 
-## First example
+The following code snippet contains everything you need to start building an app with Material UI, beginning with the `<Button>` component:
 
-The [usage page](/getting-started/usage/#quick-start) contains a small MUI example with a live editor.
-Even if you don't know anything about MUI yet, try changing the code and see how it affects the result.
+```jsx
+import * as React from 'react';
+import ReactDOM from 'react-dom';
+import Button from '@mui/material/Button';
+
+function App() {
+  return (
+    <Button variant="contained" color="primary">
+      Hello World
+    </Button>
+  );
+}
+
+ReactDOM.render(<App />, document.querySelector('#app'));
+```
+
+In the interactive demo below, try changing the code and see how it affects the output. (Hint: change `variant` to `"outlined"` and `color` to `"secondary"`. For more options, see the [`Button` component page](components/buttons/).)
+
+{{"demo": "../usage/Usage.js", "hideToolbar": true, "bg": true}}
 
 ## Example projects
 
-There are [some example projects](/getting-started/example-projects/) available, providing the infrastructure needed to develop and deploy websites in React.
+Visit the [example projects](/getting-started/example-projects/) page to see how we recommend implementing Material UI with various React libraries and frameworks like Next.js, Gatsby, Create React App, and more.
 
 ## Templates
 
-This [selection of basic templates](/getting-started/templates/) will help you get started building your app.
+Check out our [selection of basic templates](/getting-started/templates/) to get started building your next app more quickly.
 
 ## Recommended resources
 
-> ⚠️ Note that the resources linked below are using MUI v4, which is not the latest major version. As v5 has been released recently, it will take some time till new contents are created. If you have created any learning material for v5, let us know and we can link them here.
+Beyond our official documentation, there are countless members of our community who create fantastic tutorials and guides for working with Material UI.
 
-When first learning MUI, you might find third-party blog posts, books and video courses more helpful than the official documentation.
-Here are some recommended resources, some of which are free.
+The following is a curated list of some of the best third-party resources we've found—both free and paid—for learning how to build beautiful apps with our components.
+
+> **Note:** as of Spring 2022, the paid resources available all refer to outdated versions of Material UI (before v5) so unless you're using an older version, you will be more successful with the free resources.
```

Comment:
```suggestion
> **Note:** as of the second quarter of 2022, the paid resources available all refer to outdated versions of Material UI (before v5) so unless you're using an older version, you will be more successful with the free resources.
```
Making it not too Northern Hemisphere centric 😅

Reply:
Thank you!!